vcd/resource_vcd_independent_disk.go

```diff
@@ -0,0 +1,205 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"log"
+	"strings"
+)
+
+func resourceVcdIndependentDisk() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdIndependentDiskCreate,
+		Read:   resourceVcdIndependentDiskRead,
+		Delete: resourceVcdIndependentDiskDelete,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"vdc": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"storage_profile": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"size": {
+				Type:        schema.TypeFloat,
+				Required:    true,
+				ForceNew:    true,
+				Description: "size in GB",
```

Comment:
Why is size in GB as opposed to MB? GB may not be granular enough, if you say need half of GB disk only (though the use case is questionable, I agree).
One reason for using MB would be consistency, as we have `memory` and `upload_piece_size` in MB already.

Reply:
GB used as it was requested from issue. Can change to MB. Though as far as I remember I can pass values 0.5 which means I get 500MB size disk.

Reply:
I believe it may be easier for the user if we use MB everywhere consistently. For example, you have one big .tf config and there are many numbers, it's easier when you don't need to think "which parameter is in GB and which in MB".

Reply:
Done